firmware/greatfet_usb/classes/dac.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file is part of GreatFET
+ *
+ * DAC functions
+ */
+
+#include <drivers/comms.h>
+#include <stddef.h>
+#include <greatfet_core.h>
+#include <dac.h>
+#include <pins.h>
+
+#define CLASS_NUMBER_SELF (0x105)
+
+int dac_verb_set(struct command_transaction *trans)
+{
+	// set J2_P5 up as a DAC pin
+	scu_pinmux(SCU_PINMUX_GPIO2_3, SCU_GPIO_FAST | SCU_GPIO_PUP | SCU_CONF_FUNCTION0);
+	static struct gpio_t dac_pin = GPIO(2, 3);
```

Comment:
Should we be checking this pin out from the pin manager? (@ktemkin)

Reply:
Defaulting to ADC0_0 eliminates the need to use the pin manager